packages/web-components/src/components/video-player/video-player-container.ts

```diff
@@ -282,15 +283,15 @@ export const DDSVideoPlayerContainerMixin = <
      * @param videoId The video ID.
      * @internal
      */
-    _embedMedia = async (videoId: string, backgroundMode = false) => {
+    _embedMedia = async (videoId: string) => {
```

Comment:
This is probably going to be a [breaking change for CAEM](https://github.ibm.com/webstandards/carbon-for-aem/blob/main/src/components/video-player/video-player-container.ts#L94). CAEM relies on being able to call this method with this argument, as it has added `inersectionMode` property to take into consideration.

Reply:
It might be a breaking change, but I think it's one worth making and then fixing in CAEM.

In video-player-container.ts:
```ts
 _getPlayerOptions() {
      const { backgroundMode, autoPlay, muted } =
        this as unknown as DDSVideoPlayerComposite;
        // ...
}
```

Since we're passing through `intersectionMode || backgroundMode`, we should just make sure that setting `intersectionMode` to `true` also sets `backgroundMode` to `true`

Reply:
Sounds good, that should work. We'll need to remember to do that together with, or before the next C4IBM update PR in CAEM.